internal/schema/0.12/module_block.go

```diff
@@ -29,6 +29,9 @@ func moduleBlockSchema() *schema.BlockSchema {
 		},
 		Description: lang.PlainText("Module block to call a locally or remotely stored module"),
 		Body: &schema.BodySchema{
+			Extensions: &schema.BodyExtensions{
+				ForEach: true,
+			},
```

Comment:
The ticket says enable for module blocks, but 0.12 seems to not have for_each but 0.13 does. Need to check version here

Reply:
This was added in 0.13, so removing this reference